docs/compiler/Introduction.md

```diff
@@ -0,0 +1,57 @@
+<!-- README Document on how to build and use this Neural Network Compiler -->
+
+# Introduction
+
+This document discusses the ONE compiler. ONE compiler analyses, quantizes and optimizes the input model. It takes many model formats and converts to a common [Circle](#circle-format)  format which is then deployed.
```

Comment:
`analyses` ?

Reply:
`)  format ` there are two spaces

Reply:
Thanks, will fix this typo in next commit.

Reply:
@seanshpark May I know your comment regarding analyses? Do you want to remove this as ONE don't do any model analysis now?

Reply:
I don't know what kind of analyses you want to point out.  For me I don't know.